streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -279,7 +279,11 @@ public void shouldReuseSourceTopicAsChangelogs() {
 
         final InternalTopologyBuilder internalTopologyBuilder = TopologyWrapper.getInternalTopologyBuilder(builder.build());
 
-        assertThat(internalTopologyBuilder.topicGroups().get(0).stateChangelogTopics.keySet(), equalTo(Collections.singleton("topic")));
```

Comment:
This line should have told us that the original PR was not sufficient for a fix... :(